src/components/common/RowSection/RowSection.js

```diff
@@ -11,7 +13,19 @@ const RowSection = ({ title, align, books, ...props }) => {
       </Typography>
       <div className="RowSection__row">
         {books?.map((book) => (
-          <BookCard key={book._id} book={book} />
+          <BookCard
+            key={book._id}
+            book={book}
+            cardHeight="auto"
+            imageProps={{ width: '100%', height: '18rem' }}
+            buttonProps={{
+              onClick: (e) => {
+                e.preventDefault();
+                updateCart({ id: book._id, quantity: 1 });
+                toast.success('Added to Cart!!');
```

Comment:
what will happen if the request gets failed. 
You are calling the toast as success